corehq/apps/locations/views.py

```diff
@@ -164,7 +164,10 @@ def mk_loctype(name, parent_type, administrative,
             if loc_type is None:
                 loc_type = LocationType(domain=self.domain)
             loc_type.name = name
-            loc_type.administrative = administrative
+            # only CommTrack domains should have non-administrative locations
+            loc_type.administrative = (administrative
```

Comment:
(edited above, but thinking that pushing the logic to the client side would be clearest, but let me know if i'm not understanding this correctly or if that's unreasonable effort-wise)